capabilities/transformational-leadership.md

```diff
@@ -34,7 +36,7 @@ Consider the descriptions below and score yourself on the Transformational Leade
 Don't worry if the description doesn't exactly match your situation. These descriptions are meant to be examples of situations that would qualify for the associated score.
 
 1. Crisis Management: The organization is in a state of crisis or chaos, requiring leaders to take a direct and hands-on approach. Leaders focus on short-term goals, with limited opportunities to communicate a clear long-term vision or inspire team members.
-2. Transactional Leadership: Leaders prioritize meeting urgent goals and objectives, providing clear direction and guidance to team members. They begin to communicate a vision and hold team members accountable for working toward common goals.
+2. Transactional Leadership: Leaders focus on managing scenarios that deviate from the norm. They prioritize meeting urgent goals and objectives, providing clear direction and guidance to team members.They begin to communicate a vision and hold team members accountable for working toward common goals. Under this leadership style, employees are rewarded for their performance and achievements. 
```

Comment:
I deleted the last sentence here b/c it infers that supportive and transformational leadership don't reward individuals for their performance and achievements (which isn't necessarily true and might confuse survey takers).